src/ck-core/ckcheckpoint.C

```diff
@@ -214,7 +214,7 @@ public:
 };
 
 // broadcast
-void CkCheckpointMgr::Checkpoint(const char *dirname, CkCallback cb, bool _requestStatus){
+void CkCheckpointMgr::Checkpoint(const char *dirname, CkCallback _cb, bool _requestStatus){
```

Comment:
In this file, maybe the cb local variables can stay named what they are, (without underscore), since naming a local CkCallback this way is a common convention throughout the code. Instead, the global variable `cb` should be renamed to something else. I suggest `restartMainCB`.